### ISSUE TITLE
Expose Alignment interface at the top-level

### ISSUE THREAD

**Issue by nwaughachukwuma** [AskingQuestions/svelte-smooth-popover#1]
It would be nice to re-expose `Alignment` interface at the top-level.

```ts
// Reexport your entry components here
import Popover, { type Alignment } from './Popover.svelte';

export { Popover, type Alignment };
```

That way, consumers can import like so:

```svelte
import type { Alignment } from 'svelte-smooth-popover';
```

Instead of 

```svelte
import type { Alignment } from 'svelte-smooth-popover/Popover.svelte';
```

**Comment by AskingQuestions** [comment]
Thanks for the suggestion!

This is done([f013b1b](https://github.com/AskingQuestions/svelte-smooth-popover/commit/f013b1b3cefe8e02c6034c7d4054aa993076c9ec)), and published 1.0.9

**Comment by nwaughachukwuma** [comment]
Awesome! Thank you!
